.pre-commit-config.yaml

```
@@ -220,7 +220,7 @@ repos:
         types: [text]
 
       - id: validate-resolution
-        name: "'Resolution' must be a direct thread/message URL"
+        name: "'Resolution' must be a direct thread/message URL and the final header"
         language: pygrep
         entry: '(?<!\n\n)(?<=\n)Resolution: (?:(?!https://((discuss\.python\.org/t/([\w\-]+/)?\d+(/\d+)?/?)|(mail\.python\.org/pipermail/[\w\-]+/\d{4}-[A-Za-z]+/[A-Za-z0-9]+\.html)|(mail\.python\.org/archives/list/[\w\-]+@python\.org/(message|thread)/[A-Za-z0-9]+/?(#[A-Za-z0-9]+)?))\n(?=\n)))'
```

I would have liked to include more detailed descriptions of the checks in the CLI, but we also don't want to overflow the terminal width, and this one is a fair bit longer than all the other checks. As such, since (AFAIK) it isn't really any more critical that Resolution be at the end compared to any other check being in its nominal order (at least, AFAIK), IMO the better solution for PEP authors is to just drop the check for it being at the end, which avoids any extra nitpickyness to begin with.

```suggestion
        name: "'Resolution' must be a direct thread/message URL"
        language: pygrep
        entry: '(?<!\n\n)(?<=\n)Resolution: (?:(?!https://((discuss\.python\.org/t/([\w\-]+/)?\d+(/\d+)?/?)|(mail\.python\.org/pipermail/[\w\-]+/\d{4}-[A-Za-z]+/[A-Za-z0-9]+\.html)|(mail\.python\.org/archives/list/[\w\-]+@python\.org/(message|thread)/[A-Za-z0-9]+/?(#[A-Za-z0-9]+)?))\n))'
```

---

Sure, why not. 🤷🏽